bin/notify-on-additional-info-change

```diff
@@ -8,6 +8,12 @@ bin="$(dirname "$0")"
 src="${1:?A source additional info TSV file is required as the first argument.}"
 dst="${2:?A destination additional info TSV s3:// URL is required as the second argument.}"
 
+# if the file is not already present, just exit
+s3path="${2#s3://}"
+bucket="${dst_s3path%%/*}"
+key="${dst_s3path#*/}"
```

Comment:
Oh, actually, these use an undeclared variable name `dst_s3path`. I'll fix this locally and then merge.